docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,20 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Mirroring and rewriting the docker.io registry
+
+The default Docker registry `docker.io` is handled in a special way, because
+every push and pull operation gets internally suffixed with `/library` if no
+other specific namespace is defined (for example on `docker.io/username/image`).
+
+This means that a pull of `docker.io/alpine` will be internally translated to
+`docker.io/library/alpine`. A pull of `docker.io/user/alpine` will not be
+rewritten because this is already the correct remote path.
+
+This behavior has to be considered when generally rewriting registries under
+the unqualified `docker.io` URL, because containers/image has to match
+Docker’s behavior in that case.
```

Comment:
This sentence is very specific to the c/image library that many users might not be aware of (or even care about). I suggest to turn this into a more general: _"Note that the upper normalization happens to match the behavior of Docker."_